user_guide_src/source/installation/upgrade_4xx.rst

```diff
@@ -153,6 +153,12 @@ Hooks
 - Instead of CI3's ``$hook['post_controller_constructor']`` you now use
   ``Events::on('post_controller_constructor', ['MyClass', 'MyFunction']);``, with the namespace ``CodeIgniter\Events\Events;``.
 - Events are always enabled, and are available globally.
+- The hook point ``pre_controller`` and ``post_controller`` have been removed.
+  Use :doc:`../incoming/filters` instead.
+- The hook point ``display_override`` and ``cache_override`` have been removed.
+  Because the base methods have been removed.
+- The hook point ``post_system`` has moved just before sending the final rendered
```

Comment:
Did this actually move?

Reply:
What do you mean? In CI4, post_system is triggered before sending the response.
https://github.com/codeigniter4/CodeIgniter4/blob/cf43310eca93dc8edd7be3ffd86584971890628e/system/CodeIgniter.php#L554-L557

https://github.com/bcit-ci/CodeIgniter/blob/63d037565dd782795021dcbd3b1ca6f8c8a509e4/system/core/CodeIgniter.php#L494-L509

Reply:
Ohhhh I missed that this was in the CI3 transition docs! 🤦‍♂️ Sorry